cocos/gfx/base/define.ts

```diff
@@ -978,15 +975,6 @@ export class Viewport {
         this.maxDepth = info.maxDepth;
         return this;
     }
-
-    public reset (): void {
-        this.left = 0;
-        this.top = 0;
-        this.width = 0;
-        this.height = 0;
-        this.minDepth = 0;
-        this.maxDepth = 1;
```

Comment:
The public `reset` method was removed in this  PR, would it break the compatibility?

Reply:
It is newly added and might break the compatibility. But I think it should be removed.

Reply:
I will add reset back.

Reply:
reset is added back.